changelog.d/13263.bugfix

```diff
@@ -0,0 +1 @@
+Fix error when adding a User through the Synapse Admin API with a "msisdn" threepid, when "enable_email_notifs" and "email_notifs_for_new_users" are both enabled. Contributed by @thomasweston12.
```

Comment:
Small phrasing tweak:

```suggestion
Fix a bug introduced in Synapse 1.15.0 where adding a user through the Synapse Admin API with a phone number would fail if the "enable_email_notifs" and "email_notifs_for_new_users" options were enabled. Contributed by @thomasweston12.
```